shared/operation.go

```diff
@@ -106,9 +106,9 @@ func (o *Operation) GetError() error {
 	return nil
 }
 
-func (r *Operation) MetadataAsMap() (*Jmap, error) {
+func (o *Operation) MetadataAsMap() (*Jmap, error) {
 	ret := Jmap{}
-	if err := json.Unmarshal(r.Metadata, &ret); err != nil {
+	if err := json.Unmarshal(o.Metadata, &ret); err != nil {
```

Comment:
So there's probably some Go-ism here I'm not aware of, but part of your branch changes a function from using "1" as a variable to using "e". And then in this function, you change using "f" by "1".

Can you give me some context about what's that all about?

Thanks